### PR TITLE
RavenDB-15532 Support for ARM64 build

### PR DESCRIPTION
**Pull request by ayende**

### CODE REVIEW

**Comment by ayende** [comment]
@gregolsky I think we also need to update the downloads?

**Comment by ppekrol** [comment]
What are the performance numbers for this? Last time when we checked it was slower than arm32 and we decided to wait till .NET Core catches up.

**Comment by ayende** [comment]
We have users that want to run there because of memory limits, not related to performance.

**Comment by ppekrol** [comment]
libsodium, zstd, etc are working?

**Comment by ayende** [comment]
That said, import `users.dump` gives us:

image

image

This is on `a1.large` instance on AWS, 2 cores, 4 GB RAM.

8GB gp2 drive with 100 IOPS

**Comment by ayende** [comment]
image


Inserting `posts-001.dump` in 00:00:29.1865386

**Comment by ayende** [comment]
zstd works

**Comment by ayende** [comment]
Encryption & Decryption works :-)

**Comment by karmeli87** [comment]
One step closer to run on android? :-)

**Comment by gregolsky** [comment]
need to wire up on CI and other release related things

**Comment by ppekrol** [comment]
> One step closer to run on android? :-)

.NET 6 might help with that

File: build.ps1 [hunk 1]
```diff
@@ -6,6 +6,7 @@ param(
     [switch]$MacOs,
     [switch]$Osx,
     [switch]$Rpi,
+    [switch]$Arm64,
```
Comment (ppekrol) [inline build.ps1]: @gregolsky when dealing with this rename this to LinuxArm64, in .NET 5 we will have WindowsArm64 as well.

Comment (gregolsky) [reply build.ps1]: ok gotcha

**Pull request merged**